pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -859,6 +859,32 @@ func getSRIOVPCIAddresses(ifaces []v1.Interface) map[string][]string {
 	return networkToAddressesMap
 }
 
+// This function parses the NVIDIA-PASSTHROUGH-DEVICES variable that is set by
+// device plugin. It lists PCI IDs for devices allocated to the pod. The format
+// is as follows:
+//
+// "": for no allocated devices
+// "0000:81:11.1,": for a single device
+// "0000:81:11.1,0000:81:11.2[,...]": for multiple devices
+func getGpuPCIAddresses() []string {
+	pciAddrString, isSet := os.LookupEnv("NVIDIA-PASSTHROUGH-DEVICES")
```

Comment:
This too. Let's please make it generic. Please drop the company name.

Reply:
Fixed